app/api/activitylog/activitylogMiddleware.js

```diff
@@ -17,6 +17,7 @@ export const IGNORED_ENDPOINTS = [
   '/api/sync/upload',
 ];
 export const BODY_REQUIRED_ENDPOINTS = [
+  '/api/translations/import',
```

Comment:
this line is preventing to save the activity log, the body is not loaded and not needed for this implementation.

Reply:
removed